exampleCourse/courseInstances/Sp15/assessments/examplesRGallery/infoAssessment.json

```diff
@@ -0,0 +1,24 @@
+{
+    "uuid": "83730033-d2da-4361-bf29-95b3b8e8468f",
+    "type": "Homework",
+    "title": "Gallery of R Augmented Questions",
+    "set": "Homework",
+    "number": "7",
+    "allowAccess": [
+        {
+            "mode": "Public",
+            "credit": 100,
+            "startDate": "2014-07-07T00:00:01",
+            "endDate": "2024-07-10T23:59:59"
+        }
+    ],
+    "zones": [
+        {
+            "questions": [
+                {"id": "examplesRAddTwoNumbers",   "points": 2, "maxPoints": 10},
+                {"id": "examplesRGetData",   "points": 2, "maxPoints": 10},
+                {"id": "examplesRRNGSample", "points": 2, "maxPoints": 10}
```

Comment:
Also add these questions to `tests/testQuestions.js`?

Reply:
How about the `examplesRRNGSample` variant? I'm not sure all three are required.

Reply:
Good point. I'll add them.